docs/zkapps/advanced-snarkyjs/events.mdx

```diff
@@ -46,7 +46,7 @@ class MyContract extends SmartContract {
 
 Some other important facts about events:
 
-- Events are not stored on-chain. Only events from the most recent couple of transactions are retained by consensus nodes. After that, they will be discarded, but are still accessible on archive nodes. In the near future, we plan to add an API to easily fetch events from an archive node.
+- Events are not stored on-chain. Only events from the most recent couple of transactions are retained by consensus nodes. After that, the events are discarded, but are still accessible on archive nodes. 
```

Comment:
another argument for timeless documentation... let's omit references to planned features and just write about what we support now